contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/components/toolbar/FormatDropdown.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+
+  <div
+    class="dropdown-container"
+    @keydown="handleContainerKeydown"
+  >
+    <button
+      ref="dropdownButton"
+      class="format-dropdown"
+      :aria-expanded="isOpen"
+      :aria-haspopup="true"
+      :aria-label="t('textFormatOptions')"
+      @click="toggleDropdown"
+      @keydown="handleButtonKeydown"
+    >
+      <span>{{ selectedFormat }}</span>
+      <img
+        :src="require('../../../assets/icon-chevron-down.svg')"
+        alt=""
+        class="dropdown-icon"
+      >
+    </button>
+    <div
+      v-if="isOpen"
+      ref="dropdownMenu"
+      class="dropdown-menu headers-dropdown"
+      role="menu"
+      :aria-label="t('formatOptions')"
+    >
+      <div
+        v-for="(format, index) in formatOptions"
+        :key="format.value"
+        :ref="el => setItemRef(el, index)"
+        class="dropdown-item"
+        role="menuitem"
+        :tabindex="index === focusedIndex ? 0 : -1"
```

Comment:
This is definitely non-blocking feedback, but once this gets merged into unstable, it would be great for Radina (our accessibility lead) to do some testing, and potentially provide guidance about where the focus should return after making a selection in a format option (or any of the other toolbar buttons). Right now, it seems that the focus shifts to the most recently interacted line. Which makes sense! In other interactions in our Kolibri platform, we have the focus either return to, or remain on, the same button after an interaction (i.e. selecting from a dropdown menu returns the focus to the button that opened the dropdown). I am not sure the best recommendations for text editors, so we can explore that with her once we have this initial version in place for her to test!